initialize/init.go

```diff
@@ -0,0 +1,114 @@
+package initialize
+
+import (
+  "bufio"
+  "errors"
+  "os"
+
+  "github.com/craigmonson/colonize/config"
+  "github.com/craigmonson/colonize/log"
+)
+
+
+
+func Run(c *config.ColonizeConfig, l log.Logger) error {
+
+  scan := bufio.NewScanner(os.Stdin)
+  l.Log("Initializing a new Colonize project")
+
+  // environments_dir
+  l.Print("Enter 'environments_dir' [env]: ")
+  c.Environments_Dir = scan_with_default(scan, "env")
```

Comment:
Should probably move these hardcoded defaults to a struct, which will also help with the "accept all defaults" flag